django_redshift_backend/base.py

```diff
@@ -37,6 +37,7 @@ class DatabaseFeatures(BasePGDatabaseFeatures):
     has_select_for_update = False
     supports_column_check_constraints = False
     can_distinct_on_fields = False
+    allows_group_by_selected_pks = False
```

Comment:
OK.

memo:

* PostgreSQL 9.1 supports missing PK on GROUP BY: https://wiki.postgresql.org/wiki/What%27s_new_in_PostgreSQL_9.1#SQL_and_PL.2FPgSQL_features
* Django 1.9 drop support PostgreSQL 9.0 https://docs.djangoproject.com/en/1.9/releases/1.9/#dropped-support-for-postgresql-9-0
* Django ticket: https://code.djangoproject.com/ticket/19259
* Django ticket: https://code.djangoproject.com/ticket/26148